### DISCUSSION TITLE
留言 GuestBook About | Qinng

### DISCUSSION THREAD

**Discussion by qingwave** [qingwave/qingwave.github.io#66] [Announcements]
# Guestbook - Qinng

在这里留下你的足迹:)

https://qingwave.github.io/guestbook/

**Comment by whbq** [comment]
https://qingwave.github.io/ingress-real-ip/  可以转载吗？

**Comment by qingwave** [comment]
> @whbq
> https://qingwave.github.io/ingress-real-ip/  可以转载吗？
> 

可以哈，注明出处就行

**Comment by shiyajuan123** [comment]
就这？

**Comment by shiyajuan123** [comment]
惊天宝宝十分崇拜青蛙

**Comment by qingwave** [comment]
> @shiyajuan123
> 惊天宝宝十分崇拜青蛙

just so so

**Comment by shiyajuan123** [comment]
可以抄袭吗？

**Comment by lifa1225** [comment]
你好，请问这篇文章“快速实现一个Kubernetes Operator”里的代码案例可以分享下吗，我翻了下你的github没找到，感谢感谢！！！

**Comment by qingwave** [comment]
> 你好，请问这篇文章“快速实现一个Kubernetes Operator”里的代码案例可以分享下吗，我翻了下你的github没找到，感谢感谢！！！

@lifa1225 原文里有链接，源码见 https://github.com/qingwave/mygame

**Comment by lifa1225** [comment]
我最近也在研究operator，先实现mysql主从的备份和恢复，请问您那边有可以借我参考的案例吗，我google了网上都没发现合适的，感激！

**Comment by qingwave** [comment]
@lifa1225 这涉及到具体业务了，你可以github上找下，没有类似的就自己实现个

**Comment by acechef** [comment]
大佬有rss吗

**Reply by qingwave** [reply]
https://qingwave.github.io/index.xml
主题自带的有裁剪，有空我调整下

**Comment by happy-v587** [comment]
大佬，这个博客是怎么搭建的？有教程吗？

**Reply by qingwave** [reply]
目前没有教程哈，这个主题是根据[DoIt](https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/)魔改的，代码见 https://github.com/qingwave/DoIt ，配置基本与DoIt一致，如果有兴趣可以参考着试试

**Comment by freeide** [comment]
你好，兄弟，接小活吗？

**Comment by b4wu** [comment]
您好，学习go vue3。研究您的weave。在”applications“中“create appliaction"，”name"，”image“应该填什么呢？

**Reply by qingwave** [reply]
这个是指要创建docker应用(需要本地装有docker)，name随便填(唯一的就行)，image填具体镜像, 比如nginx

还有问题的话可以github提issue哈

**Comment by zbysir** [comment]
大佬的博客真好看，字也大，对我们老年人很友好。
比我的博客 https://blog.bysir.top/ 看起来更协调，真不错。

**Reply by qingwave** [reply]
:laughing: 不是啥大佬，自己瞎写着玩儿，你的博客貌似打不开呀

**Reply by zbysir** [reply]
我也发现了，很不幸，这两天貌似使用 CF CDN 的网站都被墙了。多刷新几次（好像现在好了）或者梯子试试。

refer https://www.bilibili.com/read/cv26855009/

**Reply by qingwave** [reply]
博客不错，挺有意思的

**Comment by chenyangxueHDU** [comment]
hi 你好，最近看机会吗，团队正在招聘一名高级sre，主要负责k8s稳定性相关工作，base北京，tel: 19957013491，同vx号

**Reply by qingwave** [reply]
感谢联系，暂时没有换工作的打算，欢迎多交流~

**Comment by stallonefennec** [comment]
test

**Comment by acechef** [comment]
大佬，rss失效了

**Reply by qingwave** [reply]
确实有点问题，最近都没注意，稍等我修复下

**Reply by qingwave** [reply]
依赖升级导致的，现在已恢复😄
